src/services/ApiInfo.tsx

```diff
@@ -0,0 +1,7 @@
+/* 
+ * The purpose of this class is to hold the default settings of OpenWeather API
+*/
+export default {
+    URL: "https://api.openweathermap.org/data/2.5",
+    API_KEY: "b6907d289e10d714a6e88b30761fae22",
```

Comment:
We can get these informations from .env file to not expose this kind of information in the source code.